instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/rum/RumConfigE2ETests.kt

```diff
@@ -668,6 +674,109 @@ class RumConfigE2ETests {
         sendAllRumEvents(forge, testMethodName)
     }
 
+    /**
+     * apiMethodSignature: com.datadog.android.core.configuration.Configuration$Builder#fun setVitalsUpdateFrequency(VitalsUpdateFrequency): Builder
+     */
+    @Test
+    fun rum_config_set_vitals_update_frequency_never() {
+        val testMethodName = "rum_config_set_vitals_update_frequency_never"
+        // we need to initialize the SDK on the Main Thread (Looper thread) otherwise the
+        // Choreographer Callback will not be registered
+        InstrumentationRegistry.getInstrumentation().runOnMainSync {
+            measureSdkInitialize {
+                initializeSdk(
+                    InstrumentationRegistry.getInstrumentation().targetContext,
+                    TrackingConsent.GRANTED,
+                    Configuration
```

Comment:
I would suggest either to extract `configuration` into a dedicated variable to reduce the indent level, or extract custom `viewStrategy` into a dedicated class for the same purpose.